packages/terra-avatar/src/terra-dev-site/doc/avatar/About.1.doc.mdx

```diff
@@ -17,32 +21,17 @@ This component requires the following peer dependencies be installed in your app
 | react | ^16.8.5 |
 | react-dom | ^16.8.5 |
 
-
-
 <!-- AUTO-GENERATED-CONTENT:END -->
 
 ## Usage
 
 ```jsx
-import React from 'react';
 import Avatar, { Facility, SharedUser } from 'terra-avatar';
-
-<Avatar
-  alt="User"
-  image="https://path/to/image.jpg"
-/>
-
-<Facility
-  alt="Location"
-  image="https://path/to/image.jpg"
-/>
-
-<SharedUser
-  alt="Shared-User"
-/>
 ```
 
 ## Component Features
 
  * [Cross-Browser Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#cross-browser-support)
  * [Mobile Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#mobile-support)
```

Comment:
this links seems to broken across all the components. is there any issue logged to update these links..? if not you can update these links here.

Reply:
Here's a conversation about it - https://github.com/cerner/terra-ui/issues/172#issuecomment-548389333. I think this is the right time for us to use deployed site links in place of GitHub links.

Reply:
Updated here https://github.com/cerner/terra-core/pull/2810/commits/801b0a84866f2abeb6c5187ee1b8a042b9e3d76b.